generated/nidigital/nidigital/session.py

```diff
@@ -2967,14 +3506,14 @@ def write_source_waveform_site_unique(self, waveform_name, waveform_data):
     def get_pattern_pin_names(self, start_label):
         r'''get_pattern_pin_names
 
-        TBD
+        Returns the pattern pin list. You must provide a ViChar array to serve as a buffer for the value. You pass the number of bytes in the buffer as the buffer size. If the current value of the property, including the terminating NULL byte, is larger than the size you indicate in the buffer size, the method copies (buffer size - 1) bytes into the buffer, places an ASCII NULL byte at the end of the buffer, and returns the buffer size you must pass to get the entire value. For example, if the value is "123456" and the buffer size is 4, the method places "123" into the buffer and returns 7. If you want to call this method just to get the required buffer size, you can pass 0 for the buffer size and VI_NULL for the value.
```

Comment:
Everything after the first sentence can be removed.

Reply:
Done